packages/republik-crowdfundings/graphql/schema-types.js

```diff
@@ -101,6 +101,9 @@ type PackageOption {
   additionalPeriods: [MembershipPeriod!]
 
   accessGranted: Boolean
+
+  # If true, option reward is delivered
+  postalDelivery: Boolean!
```

Comment:
hmm, I'm leaning to a more generic naming, e.g. `requireAddress`, that way we could also use it for memberships that require addresses:
```suggestion
  requireAddress: Boolean!
```

`requireAddress` would also match what we end up sending up (`address` in `PledgePaymentInput`)

Reply:
Seconding that, with this suggestion:

Moving prop into types `Goodie`, `MembershipType`. Requiring address is less of a package option flag but a reward «declaration».

This would make accessing it a level further below `packageOption.reward.requireAddress`.

Reply:
Adopted.

Reply:
Query like so:

```gql
{
  crowdfunding(name:"TEST") {
    packages {
      options {
        reward {
          ...on MembershipType {
            requireAddress
          }
          __typename
          ...on Goodie {
            requireAddress
          }
        }
      }
    }
  }
}
```